docs/userguide/storagedriver/aufs-driver.md

```diff
@@ -97,7 +97,7 @@ Storage Driver: aufs
  Dirperm1 Supported: false
 Execution Driver: native-0.2
 ...output truncated...
-````
+```
```

Comment:
Running `make docs`, I don't see any changes visually besides the removal of `sudo`.